src/screens/MemberDetail/MemberDetail.tsx

```
@@ -152,15 +173,16 @@
   // Function to handle the update of the user details
   const handleUserUpdate = async (): Promise<void> => {
     // Remove empty fields from the form state
-    function removeEmptyFields<T extends Record<string, string | File | null>>(
+    function removeEmptyFields<T extends Record<string, string | File | null | Boolean>>(
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)